Docker/techs.md

```diff
@@ -16,12 +40,12 @@ Examples of execution tuneup:
    docker run ... -v <inspection-profile.xml>:/data/profile.xml <image-name>
    ```
 
-- Save a report in HTML. The HTML report will be stored in a separate `report/` subdirectory under the `results` directory.
+- Save a report in HTML. By default, the HTML report will be stored in a separate `report/` subdirectory under the `results` directory. This location could be configured with `--report-dir`.
    ```
    docker run ... <image-name> --save-report
    ```
 
-- Display a report in HTML. After the inspection is finished, the container will not exit and will listen to port `8080`. You can connect to [`http://localhost:8080`](http://localhost:8080) to see the results.
+- Display a report in HTML. After the inspection is finished, the container will not exit and will listen to port `8080`. You can connect to [`http://localhost:8080`](http://localhost:8080) to see the results. Press Ctrl-C when done.
```

Comment:
I suggest reusing the above: "When done, you can stop the web server by pressing Ctrl-C in the Docker console."